tiff_2_png/tiff_2_png_converter.py

```diff
@@ -206,8 +214,13 @@ def convert_to_png(
                     ## check all images have the same data type
                     if bits_per_sample == "":
```

Comment:
Just adding a comment to make it clear this condition is intended for the first iteration. 
```suggestion
                    if bits_per_sample == "":  # First iteration
```

Reply:
added !